### ISSUE TITLE
A11y_Azure Extensions for VS Code_ Create New Project _keyboard: Unable to access tooltip azure function action toolbar with the help of keyboard

### ISSUE THREAD

**Issue by Yash14j** [microsoft/vscode-azurefunctions#2228]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHub Tags:#A11yTCS;#Azure Tools for VS Code;#A11y_Azure Extensions for VS Code; #DesktopApp;#Win10;#WCAG2.1.1;#WCAG1.1.1;#Benchmark;#Keyboard;#A11yMAS;#A11ySev2;#External: VSCode;

Environment Details:
Application Name: Azure Extensions for VS Code
Version: 1.46.1 (user setup)
Commit: cd9ea6488829f560dc949a8b2fb789f3cdc05f5d
Date: 2020-06-17T21:13:20.174Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Windows Version: Windows10

**Repro Steps**:

1. Open Visual studio code 
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button
4. Navigate to "Create New Project Button" from the toolbar with the help of keyboard

**Actual:**

We can see tooltip on azure function action toolbar
with the help of mouse but when we navigate the same with the help of keyboard, we do not see tooltip.
 
**Same issue is also observed with throughout the application.**

**Expected:**

Tooltip should be accessible when we navigate to azure function action toolbar with the help of keyboard

**User Impact:**

keyboard user won’t be able to know the purpose of the control as tooltip is not visible
 
**Attachment:**

[#2228.zip](https://github.com/microsoft/vscode-azurefunctions/files/4890013/2228.zip)

**Comment by EricJizbaMSFT** [comment]
This is tracked upstream here: https://github.com/microsoft/vscode/issues/52016

**Comment by Yash14j** [comment]
This issue is still repro in the Visual Studio Code version: 1.49.0, extension version 0.24.0. Please refer the attachment.
[2228_REPRO.zip](https://github.com/microsoft/vscode-azurefunctions/files/5223784/2228_REPRO.zip)

**Comment by EricJizbaMSFT** [comment]
VS Code has said they will not fix it:

> As per my comment [above](https://github.com/Microsoft/vscode/issues/52016#issuecomment-641096443) and because none of our users ever requested this I am closing this as designed. We will not change this.

Closing for that reason and due to lack of response from accessibility test team on how to get this off our tracking list.
